src/canopy_cli/cli.py

```diff
@@ -526,8 +529,8 @@ def chat(chat_server_url, rag, debug, stream):
         """
     )
 )
-@click.option("--host", default="0.0.0.0",
-              help="Hostname or ip address to bind the server to. Defaults to 0.0.0.0")
+@click.option("--host", default="localhost",
+              help="Hostname or address to bind the server to. Defaults to localhost")
```

Comment:
This particular change wasn't tested on all OSes (passing `host='localhost'` to `uvicorn.run()`).  
If you wanna merge it, we will need to test it.

Reply:
does 0.0.0.0 works?

Reply:
on all OSs - that is

Reply:
If that was the code in `main` before this PR - then yeah. I tested this code on all 3 OSes manually yesterday.